.github/workflows/n8n-dependency-update.yaml

```diff
@@ -59,7 +64,7 @@ jobs:
 
       - name: Update @types/node
         run: |
-          npm install @types/node@~${{ steps.target-versions.outputs.node_types_version }} \
+          pnpm add @types/node@~${{ steps.target-versions.outputs.node_types_version }} \
```

Comment:
I think you need to add the option `--ignore-workspace-root-check`, otherwise pnpm will fail when you execute this command in the top-level folder.

See https://pnpm.io/cli/add#--ignore-workspace-root-check

This would also affect all other `add` commands as well.